packages/graphql/src/schema/subscriptions/generate-subscription-where-type.ts

```diff
@@ -239,38 +226,30 @@ function makeRelationshipToInterfaceTypeWhereType({
     interfaceEntity,
     edgeType,
     experimental,
+    features,
+    userDefinedFieldDirectives,
 }: {
     schemaComposer: SchemaComposer;
     interfaceEntity: InterfaceEntityAdapter;
     edgeType: InputTypeComposer | undefined;
     experimental: boolean;
+    features: Neo4jFeaturesSettings | undefined;
+    userDefinedFieldDirectives: Map<string, DirectiveNode[]>;
 }): { node?: InputTypeComposer; edge?: InputTypeComposer } | undefined {
-    let interfaceImplementationsType: InputTypeComposer | undefined = undefined;
-    let interfaceNodeType: InputTypeComposer | undefined = undefined;
-
-    const implementationsFields = interfaceEntity.concreteEntities.reduce((acc, entity) => {
-        if (schemaComposer.has(entity.operations.subscriptionWhereInputTypeName)) {
-            acc[entity.name] = entity.operations.subscriptionWhereInputTypeName;
-        }
-        return acc;
-    }, {});
-    if (!isEmptyObject(implementationsFields)) {
-        interfaceImplementationsType = schemaComposer.getOrCreateITC(
-            interfaceEntity.operations.implementationsSubscriptionWhereInputTypeName,
-            (tc) => tc.addFields(implementationsFields)
-        );
-    }
-    const interfaceFields: InputTypeComposerFieldConfigMapDefinition =
-        attributesToSubscriptionsWhereInputFields(interfaceEntity);
-    if (interfaceImplementationsType && !experimental) {
-        interfaceFields["_on"] = interfaceImplementationsType;
-    }
-    if (!isEmptyObject(interfaceFields)) {
-        interfaceNodeType = schemaComposer.getOrCreateITC(
-            interfaceEntity.operations.subscriptionWhereInputTypeName,
-            (tc) => tc.addFields(interfaceFields)
-        );
-    }
+    // TODO do we really need different where-input types for subscriptions?
```

Comment:
I'm not sure this is a TODO

But to answer, yes, where input in subscriptions do not allow for relationship filtering as well as some features